### ISSUE TITLE
Blocks the app when running on Nexus 6p

### ISSUE THREAD

**Issue by arheom** [jonathanpalma/react-native-tesseract-ocr#19]
I have the following problem when running directly on my mobile phone (Nexus 6p):
- I am using the following code:
```
import RNTesseractOcr from 'react-native-tesseract-ocr';
/**
 * @param {string} imgPath - The path of the image.
 * @param {string} lang - The language you want to process.
 */
RNTesseractOcr.startOcr(imgPath, lang)
  .then((result) => {
    this.setState({ ocrResult: result });
    console.log("OCR Result: ", result);
  })
  .catch((err) => {
    console.log("OCR Error: ", err);
  })
  .done();
```
- Everything works good till the startOcr method. I have replaced the imgPath "file://" with "" and managed to have it going forward
- But, it never reaches then or catch when using with my mobile phone (Nexus 6p)
- It works well on my emulator (running Nexus 6p as well).

Any idea what to do for this? Or how can I troubleshoot? Once the issue happens, I cannot do anything on my app anymore...all links are not working. Not sure what is the problem.

Thanks!
Bogdan

**Comment by jonathanpalma** [comment]
Hey!! I'm sorry that I couldn't give support to this project in these days, but due to my new job, I'm still organizing my time to continue with this project.

If you still can't solve this issue. Is it possible you to provide the full log when the app crashes on your device?

**Comment by fazeelzama20** [comment]
@arheom Have you resolve the issue? Because i am also using same code and remove 'file://' from the image path. But it is not reaching to then part, it is catching error that OCR Error: Failed to read bitmap. Any help from your side will be appreciated..

**Comment by SwagatRanjit84** [comment]
@fazeelzama20 @arheom Did you guys solved the issue? I had same issue, file:///storage/emulated/0/documents/a.jpg is the datapath I am using but didn't work

**Comment by jonathanpalma** [comment]
I just released v2.0.0 which has many improvements that solve most of the open issues. 

Can you please git it a try and open a new issue in case there is any bug?

@arheom this might solve your issue considering that tesseract recognition is happening in a different thread

https://github.com/jonathanpalma/react-native-tesseract-ocr/blob/8bd4851edb0902a8100a33ec5f4532df0589a262/android/src/main/java/com/reactlibrary/TesseractOcrModule.java#L82-L95

@fazeelzama20 @SwagatRanjit84 you don't have to worry about removing the `file://` substring anymore

https://github.com/jonathanpalma/react-native-tesseract-ocr/blob/8bd4851edb0902a8100a33ec5f4532df0589a262/android/src/main/java/com/reactlibrary/TesseractOcrModule.java#L228
